front/src/applications/stdcm/components/StdcmHeader.tsx

```diff
@@ -18,16 +21,18 @@ const StdcmHeader = ({ isDebugMode, onDebugModeToggle }: StdcmHeaderProps) => {
           {t('notificationTitle')}
         </span>
       </div>
-      <div className="stdcm-header_debug">
-        <button
-          type="button"
-          aria-label="stdcm-debug"
-          className={cx({ 'debug-on': isDebugMode, 'debug-off': !isDebugMode })}
-          onClick={() => onDebugModeToggle(!isDebugMode)}
-        >
-          <Bug />
-        </button>
-      </div>
+      <RoleBasedContent requiredRoles={REQUIRED_USER_ROLES_FOR.FEATURES.ACCESS_STDCM_DEBUG}>
+        <div className="stdcm-v2-header_debug">
```

Comment:
(It may comes from a rebase :) )
```suggestion
        <div className="stdcm-header_debug">
```